configuration/pih/scripts/global/mch.js

```diff
@@ -27,11 +27,11 @@ function setUpEdd(currentEncounterDate, msgWeeks) {
       let yearMonthDay = lastPeriodDateValue.split('-');
       if (yearMonthDay.length == 3) {
         lastPeriodDate.setFullYear(yearMonthDay[0]);
-        lastPeriodDate.setMonth(+yearMonthDay[1] -1); // the month starts from 0 for January
+        lastPeriodDate.setMonth(+yearMonthDay[1] - 1); // the month starts from 0 for January
```

Comment:
Does this break this?  Do you need this space?

Reply:
I did not touch this code

Reply:
It's showing the line changed by you.  Just remove the extra space.

![image](https://github.com/user-attachments/assets/4263a835-c9be-4ff0-8252-b1d84f50f3dd)

Reply:
Sure . I will